src/App.js

```diff
@@ -12,15 +13,15 @@ class App extends Component {
           <HeaderNav />
           <img src={logo} className="App-logo" alt="logo" />
           <p>
-            Edit <code>src/App.js</code> and save to reload.
+            <Translate text='Edit' /> <code>src/App.js</code> <Translate text='and save to reload.' />
```

Comment:
There should be a better way to translate text with variable content, something like:

<Translate text='Edit {content}'><code>src/App.js</code></Translate>

Reply:
🤔

Reply:
Probably it is the right way:

https://www.npmjs.com/package/react-translated#styled-text

```
<Translate text='Edit `src/App.js` and save to reload.' />
```

and if you want to make it customizable:

```
<Translate
    text='Edit `{filename}` and save to reload.'
    data={{ filename: 'src/App.js' }}
/>
```